### PR TITLE
Fixed heading issue: Updated xml-instructions.md

### PR DESCRIPTION
**Pull request by jignesh-baldha**
## This PR is a:

- [ ] New topic
- [X] Content update
- [X] Content fix or rewrite
- [X] Bug fix or improvement

## Summary
Fixed heading and sidebar issue, please check here: http://nimb.ws/AaT9Rz

When this pull request is merged, it will update heading and sidebar, please check here: http://nimb.ws/6RrJFC

## Additional information
Check issue here: https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/layouts/xml-instructions.html

### CODE REVIEW

File: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Use the following layout instructions to customize your layout:
 *  <a href="#argument"><code>&lt;argument&gt;</code></a>
 *  <a href="#arguments"><code>&lt;arguments&gt;</code></a>
 
-### &lt;block&gt; {#fedg_layout_xml-instruc_ex_block}
```
Comment (osrecio) [inline guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]: I would not eliminate the "<" and the ">" in this title, it references a node inside layout.

File: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 2]
```diff
@@ -102,9 +102,9 @@ Blocks employ templates to generate {% glossarytooltip a2aff425-07dd-4bd6-9671-2
 
 To pass parameters use the <a href="#argument">`<argument></argument>`</a> instruction. 
 
-### &lt;container&gt; {#fedg_layout_xml-instruc_ex_cont}
```
Comment (osrecio) [inline guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]: I would not eliminate the "<" and the ">" in this title, it references a node inside layout.

File: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 3]
```diff
@@ -255,7 +255,7 @@ The following tables give a detailed description of the results you can get usin
    </tbody>
 </table>
 
-### &lt;action&gt; {#fedg_layout_xml-instruc_ex_act}
```
Comment (osrecio) [inline guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]: I would not eliminate the "<" and the ">" in this title, it references a node inside layout.

File: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 4]
```diff
@@ -302,7 +302,7 @@ Example:
 
 To pass parameters, use the <a href="#argument"><code>&lt;argument&gt;&lt;/argument&gt;</code></a> instruction.
 
-### &lt;referenceBlock&gt; and &lt;referenceContainer&gt; {#fedg_layout_xml-instruc_ex_ref}
```
Comment (osrecio) [inline guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]: I would not eliminate the "<" and the ">" in this title, it references a node inside layout.

File: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 5]
```diff
@@ -350,7 +350,7 @@ To pass parameters to a block use the <a href="#argument"><code>&lt;argument&gt;
     <pre>&lt;referenceContainer name="container.name" display="false" /&gt;</pre>
 </ul>  
 
-### &lt;move&gt; {#fedg_layout_xml-instruc_ex_mv}
```
Comment (osrecio) [inline guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]: I would not eliminate the "<" and the ">" in this title, it references a node inside layout.

File: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 6]
```diff
@@ -404,7 +404,7 @@ Sets the declared block or container element as a child of another element in th
    </tbody>
 </table>
 
-### &lt;remove&gt; {#fedg_layout_xml-instruc_ex_rmv}
```
Comment (osrecio) [inline guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]: I would not eliminate the "<" and the ">" in this title, it references a node inside layout.

File: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 7]
```diff
@@ -427,7 +427,7 @@ Example of usage:
 </page>
 {%endhighlight xml%}
 
-### &lt;update&gt; {#fedg_layout_xml-instruc_ex_upd}
```
Comment (osrecio) [inline guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]: I would not eliminate the "<" and the ">" in this title, it references a node inside layout.

File: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 8]
```diff
@@ -439,7 +439,7 @@ Used as follows:
 
 The specified <a href="{{ page.baseurl }}/frontend-dev-guide/layouts/layout-overview.html#handle" target="_blank">handle</a> is "included" and executed recursively.
 
-### &lt;argument&gt; {#argument}
```
Comment (osrecio) [inline guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]: I would not eliminate the "<" and the ">" in this title, it references a node inside layout.

File: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 9]
```diff
@@ -514,7 +514,7 @@ $cssClass = $this->hasCssClass() ? ' ' . $this->getCssClass() : '';
 ...
 {%endhighlight %}
 
-### &lt;arguments&gt; {#arguments}
```
Comment (osrecio) [inline guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]: I would not eliminate the "<" and the ">" in this title, it references a node inside layout.

Comment (jignesh-baldha) [reply guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]: @osrecio Could you please check https://github.com/magento/devdocs/pull/2413#issuecomment-403805452 or do you have other suggestion to fix this issue?

**Comment by jignesh-baldha** [comment]
@osrecio Could you please check https://github.com/magento/devdocs/pull/2413#issuecomment-403805452 or do you have other suggestion to fix this issue?

**Comment by osrecio** [comment]
Sorry I don't review the comment before. In that case. The PR is OK for me 👍

**Comment by jignesh-baldha** [comment]
@jeff-matthews did changes in  2.1 version of this file.

**Comment by osrecio** [comment]
Hi @jignesh-baldha , Seems like you have a lot of changes. Can you add only 1/2 commits with the unique files ?

Maybe you need update your branch with master.

**Comment by jignesh-baldha** [comment]
@osrecio Check now, I've fixed messed up code issue.

**Pull request merged**